viewer/src/main/java/rv/util/MacEnhancements.java

```diff
@@ -0,0 +1,11 @@
+package rv.util;
+
+public class MacEnhancements
+{
+	public static boolean IS_MAC = System.getProperty("os.name").contains("OS X");
```

Comment:
Should probably be final?

Reply:
Yes, my fault. That should be fixed now.